File: DevExpress-Demo/VB/RichEditDemo.Wpf/Modules/SyntaxHighlighting/SourcesCodeDocumentExporter.vb
Imports DevExpress.Office.Internal
Imports DevExpress.XtraRichEdit
Imports DevExpress.XtraRichEdit.Export

Namespace RichEditDemo

    Public Class SourcesCodeDocumentExporter
        Inherits PlainTextDocumentExporter

        Public Overrides ReadOnly Property Filter As FileDialogFilter
            Get
                Return SourcesCodeDocumentImporter._filter
            End Get
        End Property

        Public Overrides ReadOnly Property Format As DocumentFormat
            Get
                Return Id
            End Get
        End Property
    End Class
End Namespace

File: DevExpress-Demo/CS/RichEditDemo.Wpf/Modules/SyntaxHighlighting/SourcesCodeDocumentExporter.cs
using DevExpress.Office.Internal;
using DevExpress.XtraRichEdit;
using DevExpress.XtraRichEdit.Export;

namespace RichEditDemo {
    public class SourcesCodeDocumentExporter : PlainTextDocumentExporter {
        public override FileDialogFilter Filter { get { return SourcesCodeDocumentImporter._filter; } }
        public override DocumentFormat Format { get { return SourceCodeDocumentFormat.Id; } }
    }
}
